frontend/components/App/App.tsx

```diff
@@ -56,7 +57,9 @@ const App = ({ children }: IAppProps) => {
       typeof isGlobalObserver !== "undefined" &&
       !isGlobalObserver &&
       typeof isOnlyObserver !== "undefined" &&
-      !isOnlyObserver;
+      !isOnlyObserver &&
+      typeof isAnyTeamMaintainer !== "undefined" &&
```

Comment:
I believe `typeof isAnyTeamMaintainer !== undefined` is the same as `!isAnyTeamMaintainer`, right?

Reply:
I was following the pattern for isGlobalObserver and isOnlyObserver on line 56-59. Does that mean those are incorrect too?

Reply:
Not necessarily incorrect, but probably not needed since they say the same thing. See if you get the same results after removing `typeof isAnyTeamMaintainer !== "undefined"`.

Reply:
Ok, the code you're referring to is written by you in #2116  

https://github.com/fleetdm/fleet/commit/8c7155aba16812dc6a9f2ab1c63a44ea2fa15dcc#diff-6ce5edf7ea08d305614df08d6b6274d743e5edce814e2b55920775abfd2ea4bc

Just let me know if I should delete typeof for all three or what comment should I add why it's there. I don't know deepEffect well enough to have opinion or understanding why both are written but you may have some insight since you wrote it.

Reply:
!isAnyTeamMaintainer will return true if the value is false

Reply:
OOOHHH YES, my mistake. I remember now. Our permission booleans come as `undefined` unfortunately, which complicates things. So we have to deal with the fact that `false === Boolean(undefined)`. I'm sorry about that. 

In the future, we should look into removing the option of undefined.

Reply:
I'll file an issue